db/seeds.rb

```diff
@@ -5,9 +5,18 @@
                 Fabricate(:chapter_with_groups, name: "Brighton"),
                 Fabricate(:chapter_with_groups, name: "Cambridge")]
 
-    workshop = 10.times.map { |n| Fabricate(:workshop, chapter: chapters.sample, date_and_time: Time.zone.now+14.days-n.weeks) }
+    workshop = 10.times.map { |n| Fabricate(:workshop, title: 'Workshop', chapter: chapters.sample, date_and_time: Time.zone.now+14.days-n.weeks) }
+    workshop.concat 2.times.map { |n| Fabricate(:workshop, title: 'Workshop', chapter: chapters.sample) }
+
+    event = 10.times.map { |n| Fabricate(:event, name: 'Event', date_and_time: Time.zone.now+14.days-n.weeks) }
+    event.concat 2.times.map { |n| Fabricate(:event, name: 'Event') }
+
+    course = 10.times.map { |n| Fabricate(:course, chapter: chapters.sample, title: 'Course', date_and_time: Time.zone.now+14.days-n.weeks) }
+    course.concat 2.times.map { |n| Fabricate(:course, chapter: chapters.sample, title: 'Course') }
+
+    meeting = 10.times.map { |n| Fabricate(:meeting, name: 'Meeting', date_and_time: Time.zone.now+14.days-n.weeks) }
+    meeting.concat 2.times.map { |n| Fabricate(:meeting, name: 'Meeting') }
```

Comment:
Since RSVP functionality is only available for events that have not occurred yet, I needed 1 event of each type occurring at a future date. The seed file did not provide that so I updated it so it covered what I needed.